vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/action/TouchActions.java

```diff
@@ -263,6 +268,37 @@ public void performSwipe(SwipeDirection direction, int startCoordinate, int endC
         swipe(coordinates, swipeDuration);
     }
 
+    /**
+     * Performs pinch in/out by moving fingers horizontally to zoom out/in area
+     *
+     * @param pinchType type of pinch, either <b>IN</b>  to zoom out or <b>OUT</b> to zoom in
+     * @param pinchArea the area to execute the swipe
+     */
+    public void performPinch(PinchType pinchType, Rectangle pinchArea)
+    {
+        List<SwipeCoordinates> pinchCoordinates = pinchType.calculateCoordinates(pinchArea,
+                mobileApplicationConfiguration);
+        Sequence swipe1 = getSwipeSequence("finger1", pinchCoordinates.get(0));
+        Sequence swipe2 = getSwipeSequence("finger2", pinchCoordinates.get(1));
```

Comment:
why not to use mobile commands?

Reply:
Doesn't work in cases when element to zoom is overlapped by other elements